public/electron.js

```diff
@@ -0,0 +1,29 @@
+const electron = require('electron');
```

Comment:
can this file sit in the root of the project?

Reply:
When run ``npm run build`` the content of `public` is copy to `build` on production